Sources/Parley/Parley.swift

```diff
@@ -126,6 +122,8 @@ public final class Parley: ParleyProtocol {
         imageRepository = ImageRepository(messageRemoteService: messageRemoteService)
         imageRepository.dataSource = imageDataSource
         imageLoader = ImageLoader(imageRepository: imageRepository)
+
+        addObservers()
```

Comment:
Done here instead of the `configure` function otherwise the`willEnterForeground` will add more observers each time the app enters to the foreground.